### ISSUE TITLE
Typescript: Select Typescript version Command Does Nothing

### ISSUE THREAD

**Issue by izik1** [onivim/oni2#1321]
Generally a part of extensions

This _might_ be a duplicate of something, as I'm not sure what happens internally, but other than that:

*  [x] I have checked the issue tracker for duplicate issues.

Note that this is _not_ the same as #538. (that's talking about `:version`, this is about the one from the Typescript plugin.)

**Comment by glennsl** [comment]
Blocked on missing the UI primitives needed to hook this up.

**Comment by bryphe** [comment]
Thanks for logging, @izik1 !

We now have the quick-input API hooked up from the extension host - so selecting typescript version now gives a menu:
![image](https://user-images.githubusercontent.com/13532591/88595271-81e03080-d017-11ea-83fb-105416c79bfd.png)

Implemented in https://github.com/onivim/oni2/pull/2148
